src/main/java/com/devoxx/genie/action/AddDirectoryAction.java

```diff
@@ -58,7 +58,7 @@ private void addDirectoryToContext(Project project, @NotNull VirtualFile directo
         if (!filesToAdd.isEmpty()) {
             fileListManager.addFiles(filesToAdd);
 
-            ModelProvider selectedProvider = ModelProvider.fromString(settings.getSelectedProvider(project.getLocationHash()));
+            ModelProvider selectedProvider = ModelProvider.fromString(settings.getSelectedProvider(project.getLocationHash())); //todo test
```

Comment:
the TODO is still relevant? If tested, you can remove it.

Reply:
Tested and removed now, thanks!